app/assets/javascripts/tag_by_tag_plate.js

```diff
@@ -66,17 +76,34 @@
         };
       },
       validators: [
+        // `t` is a qcLookup object
+        // The data for t.template comes from app/models/labware_creators/tagging/tag_collection.rb
```

Comment:
Yes, sorry about this short name. I think it was supposed to be short for 'this'.